packages/ethereum/Dockerfile.hardhat

```diff
@@ -0,0 +1,57 @@
+ARG HOPR_TOOLCHAIN_IMAGE=${HOPR_TOOLCHAIN_IMAGE:-hopr-toolchain}
+
+# Run HOPRd  within a single container using yarn
+FROM ${HOPR_TOOLCHAIN_IMAGE} as build
+
+WORKDIR /app/hoprnet
+
+# Get all we need from the monorepo
+COPY .yarn .yarn/
+COPY packages packages/
+COPY package.json .yarnrc.yml yarn.lock tsconfig.json Makefile Makefile.deps Cargo.toml Cargo.lock ./
+
+LABEL description="Launches a hardhat node running a local network with the HOPR contracts deployed and available."
+
+WORKDIR /app/hoprnet/packages/ethereum
```

Comment:
Switching workdirs so often isn't very commonly done. Instead use `cd` in the `RUN` lines.

Reply:
changed